pkg/controller/autoscaler/tidbcluster_autoscaler_control.go

```diff
@@ -42,9 +47,8 @@ func (tac *defaultAutoScalerControl) ResconcileAutoScaler(ta *v1alpha1.TidbClust
 	return errors.NewAggregate(errs)
 }
 
-//TODO: sync autoScaler
 func (tac *defaultAutoScalerControl) reconcileAutoScaler(ta *v1alpha1.TidbClusterAutoScaler) error {
-	return nil
+	return tac.autoScalerManager.Sync(ta)
```

Comment:
`ta` should be a DeepCopy?

Reply:
Why?

Reply:
DeepCopy in the beginning of the sync logic then you do not need to care whether you will update that resource in following logic, anyway, it's just a suggestion.